lib/file_generator.dart

```diff
@@ -363,7 +363,7 @@ class FileGenerator extends ProtobufContainer {
       // Make sure any other symbols in dart:core don't cause name conflicts
       // with enums that have the same name.
       out.println("// ignore: UNUSED_SHOWN_NAME\n"
-          "import 'dart:core' show int, dynamic, String, List, Map;");
```

Comment:
We added `dynamic` since otherwise dart2js would fail on the generated code.

Unfortunately, dart2js and the analyzer disagree on whether `dynamic` is a part of `dart:core`, and I don't think this has been fixed yet: https://github.com/dart-lang/sdk/issues/29125.

So we probably still need this one, even though it generates an analyzer warning -- unless you've verified the dart2js problem is gone?